docs/source/developers/cpp/building.rst

```diff
@@ -15,6 +15,8 @@
 .. specific language governing permissions and limitations
 .. under the License.
 
+.. highlight:: console
```

Comment:
This sets the default for the full file. Alternative is specifying it for each code-block (like we now had 'shell' for each code block). The latter might actually be more explicit, so either way is fine for me.

Reply:
Are there other files where we should apply this? (or switch "shell" code blocks to "console"?)

Reply:
If there is a need to make a code block in another language I assume it supersedes the `console` set for the full file?

Reply:
> If there is a need to make a code block in another language I assume it supersedes the console set for the full file?

Yes, indeed